app/services/workflow_reporter.rb

```diff
@@ -0,0 +1,25 @@
+
+# send errors to preservationAuditWF workflow for an object via ReST calls.
+class WorkflowReporter
+
+  DOR = 'dor'.freeze
+  PRESERVATIONAUDITWF = 'preservationAuditWF'.freeze
+  NO_WORKFLOW_HOOKUP = 'no workflow hookup - assume you are in test or dev environment'.freeze
+  # this method will always return true because of the dor-workflow-service gem
```

Comment:
very nitpicky, but i'd sorta like a blank line between the last constant and the method comment. mildly surprised rubocop didn't complain.